src/ConfigFileReader.php

```diff
@@ -0,0 +1,93 @@
+<?php
+declare(strict_types=1);
+
+namespace wearewondrous\PshToolbelt;
+
+use Robo\Config\Config as RoboConfig;
+use Robo\Robo;
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
+
+class ConfigFileReader {
+
+  const SRC_PATH = 'vendor/wearewondrous/psh-toolbelt/src';
+
+  /**
+   * @var RoboConfig
+   */
+  protected $roboConfig;
+
+  /**
+   * @var string
+   */
+  private $rootDirectory;
+
+  /**
+   * SiteSettings constructor.
+   *
+   * @param string $rootDirectory
+   */
+  public function __construct(string $rootDirectory = null) {
+    if(!$rootDirectory) {
+      $rootDirectory = $this->getRootDir();
+    }
+
+    $this->rootDirectory = $rootDirectory;
+
+    $this->roboConfig = $this->getRoboConfig();
+  }
+
+  /**
+   * Get merged yml config.
+   *
+   * @return RoboConfig
+   */
+  public function getRoboConfig(): RoboConfig {
```

Comment:
Actually, no function here needs to be part of the Class. I mean, every function can well be also a static one...

Reply:
Not true as i need the class to be OOP style because of testing and the injection of the filesystem